### PR TITLE
Cs fixes1

### PR DESCRIPTION
**Pull request by beberlei**

### CODE REVIEW

**Review by greg0ire — COMMENTED** [review]
I think we should postpone the `_doExecute()` rename to at least 2.9.x (or 3.0)

**Comment by beberlei** [comment]
You are right, slightly more defensive is good

**Comment by SenseException** [comment]
> I think we should postpone the _doExecute() rename to at least 2.9.x (or 3.0)

Isn't that renaming connected to a new deprecation? Otherwise it will break on an 2.8 -> 2.9 update. I'd expect `_doExecute()` completely disappear in 3.0.

File: lib/Doctrine/ORM/Query.php [hunk 1]
```diff
@@ -751,7 +757,7 @@ public function getLockMode()
      *
      * @return string
      */
-    protected function _getQueryCacheId()
+    protected function getQueryCacheId()
```
Comment (greg0ire) [inline lib/Doctrine/ORM/Query.php:760]: Ah we have another BC-break here

**Review by greg0ire — APPROVED** [review]

**Pull request merged**